README.md

```diff
@@ -1,2 +1,32 @@
 # rex
+
 Simple utilities for Deno superpowers
+
+## Usage
+
+To use this library, import individual functions from the respective files. Because there's such a
+wide variety of functionality, it's broken up into several folders and files within those folders.
+
+For example:
+
+```typescript
+import { isNil } from 'https://deno.land/x/rex/lang/values.ts';
+```
+
+Or:
+
+```typescript
+import { $ } from 'https://deno.land/x/rex/shell/run.ts';
+```
+
+## TODOs
+
+- Improve documentation quality
```

Comment:
Sorry, I'm going to be nitpicky about the TODO's list here - I feel especially in the open source world that it's really valuable to provide specifics in these sorts of lists. What kind of documentation would you like to see? This is helpful for building the community around a library and fostering discussion. 'Improve documentation quality' is something you could say about almost literally anything, regardless of how well it's already documented. Specifics like 'add example usages for each function' is something that's actionable and someone might want to take up!

Reply:
The inline docs are excellent by the way, I think one specific documentation TODO could be to add an index or listing of the functions in the Readme for easy perusal.

Reply:
Good call! Hopefully this is better now.